contracts/package.json

```diff
@@ -19,7 +19,7 @@
     "fund": "FORK=true npx hardhat fund --network localhost",
     "copy-interface-artifacts": "mkdir -p ../dapp/abis && cp artifacts/contracts/interfaces/IVault.sol/IVault.json ../dapp/abis/IVault.json && cp artifacts/contracts/liquidity/LiquidityReward.sol/LiquidityReward.json ../dapp/abis/LiquidityReward.json && cp artifacts/contracts/interfaces/uniswap/IUniswapV2Pair.sol/IUniswapV2Pair.json ../dapp/abis/IUniswapV2Pair.json && cp artifacts/contracts/staking/SingleAssetStaking.sol/SingleAssetStaking.json ../dapp/abis/SingleAssetStaking.json",
     "echidna": "yarn run clean && echidna-test . --contract PropertiesOUSDTransferable --config contracts/crytic/TestOUSDTransferable.yaml",
-    "slither": "yarn run clean && slither . --filter-paths \"crytic|mocks|@openzeppelin\" --exclude-low --exclude-informational --exclude conformance-to-solidity-naming-conventions,different-pragma-directives-are-used,external-function,assembly",
+    "slither": "yarn run clean && slither . --filter-paths \"crytic|mocks|@openzeppelin\" --exclude-low --exclude-informational --exclude conformance-to-solidity-naming-conventions,different-pragma-directives-are-used,external-function,assembly,controlled-array-length,weak-prng,divide-before-multiply",
```

Comment:
These rules may uncover issues in the future on other contracts.
So I wonder if, as opposed to excluding them globally, we should add the known false positives they return to [slither.db.json](https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/slither.db.json)?
I'm actually not very familiar with the best way to update slither.db.json - would be good to get input on that from @shahthepro or @tomlinton (and we should also document it in our security repo so that everyone knows).

Reply:
From what I can tell the `slither.db.json` is not something that was intended to be user editable.

The slither command does support the `--filter-paths` parameter. We could use that to run slither multiple times on different files with different `--exclude` parameters. This way we could single out the source files that have false positives and run slither with exclusion of specific detectors only on those. 

Though I am not completely thrilled by that idea, since it is going to take more time to perform the runs and can become really messy maintenance wise.

Reply:
You can run `yarn run slither --triage` to regenerate the slither DB

Reply:
Nice thanks @shahthepro will look into that tomorrow